source/config/201-clusterrole.yaml

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: eventing-sources-rabbitmq-controller
+  labels:
+    contrib.eventing.knative.dev/release: devel
+rules:
+
+  - apiGroups:
+      - sources.knative.dev
```

Comment:
should projects in the sandbox continue to use the common api groups or define their own?
cc @vaikas @evankanderson

Reply:
Good question. There's been some [discussion about that in the TOC and Steering](https://docs.google.com/document/d/1e0PpX4uUaaolF7k5p05ZK_ScjpS5VRiK5y8cCg4oH7Q/edit#heading=h.ya000o6lypo9); the current proposal is to use a separate domain for the CRD APIs, but I don't think we have such a subdomain yet.

In this case, I'd suggest begging forgiveness and alignment with the other sources.